includes/Renderer/Stories/Renderer.php

```diff
@@ -624,6 +624,21 @@ protected function render_story_with_poster() {
 		}
 	}
 
+	/**
+	 * Allows customization of html attributes in the web stories widget anchor tag loop
+	 * Converts array into escaped inline html attributes
+	 *
+	 * @return void
+	 */
+	protected function render_attributes() {
+		$story_render_attributes = apply_filters( 'story_render_attributes', [], $this->current(), $this->position, $this->get_view_type() );
+		if ( ! empty( $story_render_attributes ) ) {
+			foreach ( $story_render_attributes as $attribute => $value ) {
+				echo ' ' . esc_attr( $attribute ) . '="' . esc_attr( $value ) . '"';
```

Comment:
`esc_attr()` should only be used for attribute values, not attribute names.

I think we could use `wp_kses_one_attr()` to escape the output here:

```suggestion
				echo ' ' . wp_kses_one_attr( $attribute . '="' . esc_attr( $value ) . '"' );
```

Reply:
Thank you, I actually didn't know about this method, very useful!

Reply:
@swissspidy I was just checking this, it seems like we need a second param as the element we are adding it for.
`echo ' ' . wp_kses_one_attr( $attribute . '="' . esc_attr( $value ) . '"', 'a' );`

I was just testing to see if it would render it correctly and it does! 
I'm getting _phpcs_ complaints on my end, I think that's why I ended up going for `esc_attr` on the attribute name.

Are you happy to add as it's above? I will wait on your response before I commit.

Reply:
fyi I went ahead and committed the changes!